eng/testing/tests.mobile.targets

```diff
@@ -195,5 +192,5 @@
   <Target Name="PublishTestAsSelfContained"
           Condition="'$(IsCrossTargetingBuild)' != 'true'"
           AfterTargets="Build"
-          DependsOnTargets="Publish;BundleTestAppleApp;BundleTestAndroidApp;BundleTestWasmApp;ArchiveTests" />
+          DependsOnTargets="Publish;BundleTestAppleApp;BundleTestAndroidApp;WasmBuildApp;ArchiveTests" />
```

Comment:
It looks like you are only including the targets file on Browser to the WasmBuildApp target doesn't exist on other platforms and things break.

Reply:
Yikes! Hopefully, fixed with the new commit.